packages/edit-post/src/components/layout/index.js

```diff
@@ -76,6 +77,7 @@ function Layout( {
 			<BrowserURL />
 			<UnsavedChangesWarning />
 			<AutosaveMonitor />
+			<LocalAutosaveMonitor />
```

Comment:
Would it make sense to combine some of these monitors?

Reply:
I wonder as well. Would it make sense to only save locally if saving otherwise fails?

Reply:
> Would it make sense to combine some of these monitors?

As implemented, the `LocalAutosaveMonitor` is effectively an extension of the base `AutosaveMonitor` behavior.

Reply:
> I wonder as well. Would it make sense to only save locally if saving otherwise fails?

We could. The local autosave occurs more frequently than the server-persisted autosave (by default at least), so I'd assume there's technically a potential for data loss in that ~45s gap. I dunno that it's worth the extra complexity to determine whether a failure occurred.

Reply:
Yes, let's save locally regardless of remote saving. Plus, now that we are to switch to `sessionStorage`, we have no real reason to worry about wasting storage.